frontend/integration-tests/tests/login.scenario.ts

```diff
@@ -40,24 +40,21 @@ describe('Auth test', () => {
     });
 
     it('does not show admin nav items in Administration to htpasswd user', async() => {
+      // Let flags resolve before checking for the presence of nav items.
+      await browser.sleep(5000);
```

Comment:
I'm not crazy about this, but I can't think of another way. It's hard to wait for something to not show up.